aries-site/src/pages/foundation/branding.js

```diff
@@ -10,6 +17,36 @@ const Branding = () => (
     <ContentSection>
       <MainHeading>{title}</MainHeading>
     </ContentSection>
+    <ContentSection>
+      <Subsection name="HPE Element">
+        <UsageExample
+          label="hpe-element"
+          themeMode="light"
+          pad={{ horizontal: 'large', vertical: 'small' }}
+        >
+          <Box direction="row" align="center" gap="medium">
+            <Hpe color="brand" size="xlarge" />
+            <Box direction="row" gap="xsmall">
+              <Text weight="bold">HPE</Text>
+              <Text>Service Name</Text>
+            </Box>
+          </Box>
```

Comment:
I'm thinking this behavior is duplicated and can be its own separate component. (see lines 40-46)

Reply:
Yes, it is duplicated. I don't think it will be used other than these two instances, but I can add it as a component local to `branding.js`.

Reply:
Done.